tests/playbooks/kubevirt_preset.yml

```diff
@@ -0,0 +1,34 @@
+- hosts: localhost
+  gather_facts: false
+  connection: local
+
+  tasks:
```

Comment:
Can we use [example](https://github.com/kubevirt/kubevirt/blob/master/cluster/examples/vmi-preset-small.yaml) from kubevirt repo?

Reply:
done